sdfformat/install.bash

```diff
@@ -13,4 +13,9 @@ else
     tue-install-debug "Gazebo source has already been added"
 fi
 
-tue-install-system libsdformat4-dev
+if [ "$(lsb_release -sc)" == "xenial" ]
```

Comment:
How about older version?

Reply:
We anyway don't support versions older than xenial so why include it?